### ISSUE TITLE
command line call does not work

### ISSUE THREAD

**Issue by GitBiker** [oliverschwendener/ueli#125]
>"C:\Program Files\Mozilla Firefox\firefox.exe"
does not work

**Comment by oliverschwendener** [comment]
Can you explain in more detail what you are trying to do and what doesn't work?

**Comment by GitBiker** [comment]
I expect Firefox to load like it does in the normal command window.

**Comment by oliverschwendener** [comment]
And what is your exact input?

**Comment by GitBiker** [comment]
oh, I'm sorry. There is an error in my first comment.
I use:
>"C:\Program Files\Mozilla Firefox\firefox.exe"

**Comment by GitBiker** [comment]
ok, markdown styling is formatting my text again.
This is my input:
ueli command character"C:\Program Files\Mozilla Firefox\firefox.exe"

**Comment by oliverschwendener** [comment]
so this?

```
>C:\Program Files\Mozilla Firefox\firefox.exe
```

**Comment by GitBiker** [comment]
no, with double quotes around the path.

**Comment by oliverschwendener** [comment]
And what is the error message you get?

**Comment by GitBiker** [comment]
nothing happens, nothing is shown.

**Comment by oliverschwendener** [comment]
Ok, I'll look into this.

Next time please be more precise with your issue description (steps to reproduce the bug/what do you expect to happen/what actually happend, version number you are using, maybe screenshots).

**Comment by GitBiker** [comment]
you're welcome.

**Comment by oliverschwendener** [comment]
In the meantime: if you want to open firefox why don't you just enter the filepath?

```
C:\Program Files\Mozilla Firefox\firefox.exe
```

**Comment by GitBiker** [comment]
this is what I actually want to do:
cmd "C:\Program Files\Everything\Everything.exe" -instance "ETP Server"

**Comment by oliverschwendener** [comment]
This should be fixed with the newset [v8.0.0](https://github.com/oliverschwendener/ueli/releases/v8.0.0) release. Please reopen this issue if this is not the case.
